source/develop/release-management/features/virt/zerocopy-migrations.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Zero-copy migrations
+category: feature
+authors: mzamazal
+---
+
+# Zero-copy migrations
+
+## Summary
+
+This feature adds support for zero-copy migrations.
+
+## Owner
+
+*   Name: Milan Zamazal
+*   Email: mzamazal@redhat.com
+
+## Description
+
+Zero-copy migrations further improve migration speed and may allow migrating large VMs (>1 TB RAM).
+
+Zero-copy migrations can be enabled by passing ``VIR_MIGRATE_ZEROCOPY`` flag to [virDomainMigrateToURI3](https://libvirt.org/html/libvirt-libvirt-domain.html#virDomainMigrateToURI3) libvirt call.
+
+## Prerequisites
+
+Nothing special, just up-to-date libvirt and QEMU.
+
+## Limitations
+
+Current implementation of zero-copy migration in QEMU has some restrictions:
+
+- It can be used only with [parallel migrations](parallel-migrations.html) and thus the same limitations as for parallel migrations apply.
+
+- It cannot be used with migration encryption.
+
+- QEMU must be permitted to lock guest RAM pages in memory (libvirt takes care of this unless memory limits are set manually in ``<memtune>``.
+
+- This is a new feature in QEMU/libvirt and may contain bugs.  It's not recommended to use it in oVirt at the moment it unless it is really needed.
+
+- Vdsm, libvirt and QEMU on the migration source host must support the feature. 
+
+## User experience
+
+A new migration policy *Very large VMs* is introduced in cluster migration options.  If this policy is selected, zero-copy migrations are conditionally enabled and other migrations parameters specified in the policy are applied (this allows using migrations parameters such as downtime schedule customized for such migrations).  The policy is available only when parallel migrations are enabled in Engine config (i.e. on cluster level 4.7 by default).
+
+If this policy is selected while the [limitations above](#limitations) are not satisfied then:
+
+- If parallel migrations are disabled or only conditionally enabled ("Auto") then they are enabled with automatic number of parallel connections.
+
+- If migration encryption is enabled then zero-copy is not enabled.
+
+If the policy is selected and the host doesn't support the feature then:
+
+- If Vdsm doesn't support the feature, API call mismatch is logged and zero-copy is not enabled.
+
+- If Vdsm supports the feature but libvirt doesn't then zero-copy is not enabled.
```

Comment:
I don't think you have to handle this case. You can bump requirements as part of this feature

Reply:
Right, the feature should reach RHEL and CentOS Stream 8 eventually.  Removed.